openapi/controllers/main.py

```diff
@@ -39,3 +49,77 @@ def copy(obj, **d):
         }})
 
         return result
+
+
+class OAS(http.Controller):
+
+    @http.route('/api/v1/<namespace_name>/swagger.json',
+                type='http', auth='public', csrf=False)
```

Comment:
We cannot give everyone access to a spec file.
Here should be auth='user', csrf=True(it's default)

Reply:
Problem is, that to work with swagger UI, it needs to access this link directly. Only possible workaround, I could think of: Use odoo's of token authentication and generate `?token=...` links (not sure about the exact parameters in odoo). Similar to how one time access is granted to sales document.